modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/util/table-columns.tsx

```diff
@@ -711,6 +711,12 @@ export const metricsListColumns = {
 		label,
 		sortable: false
 	}),
+	impressionMadeMetric: {
+		accessor: 'impressionMadeMetric',
+		className: 'table-column-text-end',
+		dataFormatter: data => data.toLocaleString(),
+		label: Liferay.Language.get('impressions')
+	},
```

Comment:
It cannot be sorted because it affects the columns in the UI.

Reply:
@CaioERP please pay attention in this comment when sending this PR. thanks